frontend/src/old-pages/Clusters/Properties.tsx

```diff
@@ -100,36 +100,25 @@ export default function ClusterProperties() {
                     triggerType="custom"
                     content={
                       <StatusIndicator type="success">
-                        ARN copied
+                        SSH Command copied
```

Comment:
chore: we are moving towards a more robust product and started migrating all hard-coded labels to an i18n framework so the copywriting will be easier to review and translate. That's why I suggest you move all the edited labels to the `strings.json` file and use the `t` function from the `useTranslation` hook to retrieve the values.
An example of how this is done in [this commit](https://github.com/aws-samples/pcluster-manager/commit/a2e2adbc70bedc675457fe4ce6ca74585e10328d).

Reply:
Thanks I'll do that!

Reply:
done ✅